microsoft-excel-online/1.0/modules/ROOT/pages/microsoft-excel-online-connector-reference.adoc

```diff
@@ -0,0 +1,4921 @@
+* <<GetColumnsBeforeRange>>
+* <<GetNamedItem>>
+* <<GetOffsetRange>>
+* <<GetRange>>
+* <<GetRangeCell>>
+* <<GetRangeColumn>>
+* <<GetRangeEntireColumn>>
+* <<GetRangeEntireRow>>
+* <<GetRangeFormat>>
+* <<GetRangeIntersection>>
+* <<GetRangeLastCell>>
+* <<GetRangeLastColumn>>
+* <<GetRangeLastRow>>
+* <<GetRangeRow>>
+* <<GetRangeVisibleView>>
+* <<GetResizedRange>>
+* <<GetRow>>
+* <<GetRowsAboveRange>>
+* <<GetRowsBelowRange>>
+* <<GetTable>>
+* <<GetTableRange>>
+* <<GetUsedRange>>
+* <<GetWorkbookApplication>>
+* <<GetWorkbookComment>>
+* <<GetWorksheet>>
+* <<InsertRange>>
+* <<ListChartSeries>>
+* <<ListCharts>>
+* <<ListColumns>>
+* <<ListNamedItem>>
+* <<ListNames>>
+* <<ListPivotTables>>
+* <<ListRows>>
+* <<ListTables>>
+* <<ListWorkbookComments>>
+* <<ListWorksheet>>
+* <<RangeBoundingRect>>
+* <<RangeClear>>
+* <<RangeDelete>>
+* <<RangeMerge>>
+* <<RangeUnmerge>>
+* <<RefreshSession>>
+* <<ResetChartData>>
+* <<Unauthorize>>
+* <<UpdateChart>>
+* <<UpdateNamedItem>>
+* <<UpdateRange>>
+* <<UpdateRangeFormat>>
+* <<UpdateRow>>
+* <<UpdateTable>>
+* <<UpdateWorksheet>>
+
+== Associated Sources
+* <<NewRowListener>>
+* <<NewWorksheetListener>>
+* <<UpdatedRowListener>>
+
+
+== Operations
+
+[[AddChart]]
+== Add Chart
+`<excel:add-chart>`
+
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Workbook Id a| String |  The ID of the workbook. |  | x
+| Worksheet Name a| String |  The name of the worksheet. |  | x
+| Chart Body a| Any |  The body of the chart to add. |  #[payload] |
+| Config Ref a| ConfigurationProvider |  The name of the configuration to use to execute this component. |  | x
+| Streaming Strategy a| * <<RepeatableInMemoryStream>>
+* <<RepeatableFileStoreStream>>
+* non-repeatable-stream |  Configures how Mule processes streams. Repeatable streams are the default behavior. |  |
+| Response timeout a| Number |  Response timeout for the request. The ones specified here will overwrite the default ones specified in the config. |  |
+| Response Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Response timeout unit for the request. The ones specified here will overwrite the default ones specified in the config. |  |
+| Custom headers a| Object |  Custom headers that the request will include. The ones specified here will overwrite the default ones specified in the config. |  |
+| Custom query parameters a| Object |  Custom query parameters that the request will include. The ones specified here will overwrite the default ones specified in the config. |  |
+| Target Variable a| String |  Name of the variable that stores the operation's output. |  |
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable. |  #[payload] |
```

Comment:
Global only if you have time: Replace text with this text from our Reference description spreadsheet: 

Expression that evaluates the operation's output. The expression outcome is stored in the target variable.